Sources/Modules/EternalResolve/Items/Weapons/StabbingSwords/Projectiles/StabbingProjectile.cs

```diff
@@ -137,6 +137,11 @@ public override void AI()
 			UpdateDarkDraw();
 			UpdateLightDraw();
 		}
+		public virtual void HitTileSound(float scale)
+		{
+			SoundEngine.PlaySound((SoundID.NPCHit4.WithVolume(1 - scale / 2.42f)).WithPitchOffset(Main.rand.NextFloat(-0.4f, 0.4f)), Projectile.Center);
```

Comment:
It is a bit too loud for me.
```suggestion
			SoundEngine.PlaySound((SoundID.NPCHit4.WithVolume(0.5f - scale / 2.42f)).WithPitchOffset(Main.rand.NextFloat(-0.4f, 0.4f)), Projectile.Center);
```